lib/config/index.js

```diff
@@ -74,6 +74,71 @@ exports.getLoaderConfig = function() {
   return cfg;
 };
 
+function updateNodelibTarget(target) {
+  target.registry = 'npm';
+  target.name = target.name.replace('github:jspm/nodelibs-', 'npm:jspm-nodelibs-');
+  target.package = target.package.replace('jspm/nodelibs-', 'jspm-nodelibs-');
+  target.exactPackage = target.exactPackage.replace('jspm/nodelibs-', 'jspm-nodelibs-');
+  target.exactName = target.exactName.replace('github:jspm/nodelibs-', 'npm:jspm-nodelibs-');
+  target.exactNameEncoded = target.exactNameEncoded.replace('github:jspm/nodelibs-', 'npm:jspm-nodelibs-');
+  target.setVersion('0.2.0');
+  return target;
```

Comment:
Rather than this manual modification, we can use `target.setVersion`, `target.setRegistry` and `target.setPackage`.